### DISCUSSION TITLE
Resources to make plugins?

### DISCUSSION THREAD

**Discussion by kalinkochnev** [alainm23/planify#671] [Q&A]
I just started using planner today and I'm absolutely in love. I saw that there is a plugin system, but it doesn't seem like there are any resources to start making them. Is this currently in the works? Thanks.

**Comment by BeatLink** [comment]
Plugins are quite a new feature but its so exciting to see users such as yourself willing to create and contribute to developing them! I'm sure @alainm23 would be happy to explain the plugin architecture for you to get started!

**Comment by koudi** [comment]
Try looking at source for [current plugins](https://github.com/alainm23/planner/tree/master/plugins). Looks like you get access to database (and maybe other services) and handle to main window + left sidebar. I too wish there was more info, but I guess this is still very fresh and @alainm23  probably hasn't settled on the final form yet.
